### ISSUE TITLE
Issues raised from blocked users cannot be seen or closed

### ISSUE THREAD

**Issue by dinglemouse2250** [Codewars/codewars.com#1840]
Seems to have been previously reported in related #808 (which is now closed?)

If a blockee raises a Kata issue, that issue comment cannot be seen by the blocker

Which causes 2 problems:
* the issue cannot be fixed by the blocker (because don't know what it is saying)
* the issue cannot be marked resolved by the blocker (because unable to reply to the comment)

![image](https://user-images.githubusercontent.com/22114719/59260982-f8896580-8c7f-11e9-8715-9157f8d7d2aa.png)

**Comment by hobovsky** [comment]
So basically "I do not want to hear from this guy, but I wonder what he has to say"? ;)

**Comment by dinglemouse2250** [comment]
> So basically "I do not want to hear from this guy, but I wonder what he has to say"? ;)

"I do not want to hear from this guy" - in general, Yes
"but I wonder what he has to say" - but only for *Issues* because I still need a way to fix valid problems.

**Comment by Voileexperiments** [comment]
Yeah but to be honest I don't have any hope whatsoever that #1365 will be resolved in general.

Or of course, just don't use the block function. Then there is no need to deal with all the potential problems that come with this "feature"!

**Comment by Blind4Basics** [comment]
> So basically "I do not want to hear from this guy, but I wonder what he has to say"? ;)

the thing is, Voile should never have been able to post those issues in the first place, since he's blocked. So, I guess that's the "end points" thing where the blocking thin isn't enforced.

**Comment by Voileexperiments** [comment]
Yes, but then again, that's because #1365 is unresolved for a long time. I prefer preserving the beta process more than preserving a hastily slapped out "feature" that hasn't really done anything particular useful.

Besides, it's not like it's effective at all, you know right? If I *really* had to open an issue I can just post it with an alt account. Ultimately it's, again, the feature's fault, I'm just messenger, you know :wink:
